imports/collections/onlineusers_private.js

```diff
@@ -0,0 +1,38 @@
+import { Meteor } from 'meteor/meteor';
+import { OnlineUsersSchema } from './onlineusers.schema';
+import moment from 'moment/moment';
+
+if (Meteor.isServer) {
+    Meteor.publish('onlineUsersForRoute', function (route) {
+        return OnlineUsersSchema.find({activeRoute: route});
+    });
+}
+
+const checkRouteParamAndAuthorization = (route, userId) => {
+    check(route, String);
+    if (!userId) {
+        throw new Meteor.Error('not-authorized');
+    }
+};
+
+Meteor.methods({
+    'onlineUsers.enterRoute'(route) {
+        const userId = Meteor.userId();
+        checkRouteParamAndAuthorization(route, userId);
+
+        OnlineUsersSchema.insert({
+            userId: userId,
+            activeRoute:route,
+            updatedAt: new Date()
+        });
+
+        // remove outdated entries
+        const fiveMinAgo = moment().add(-1,'minutes').toDate();
```

Comment:
Ups. Thanks! I updated the variable name (I think a minute should be fine). Additionally I added a further feature: The client now updates the connection state 30 seconds. If a user just follows the changes of a minutes he will be on this route without doing anything except scrolling and reading.